src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinInetProxyHelper.cs

```diff
@@ -154,6 +178,16 @@ public bool ManualSettingsOnly
                     }
                     else
                     {
+                        if (lastError == Interop.WinHttp.ERROR_WINHTTP_AUTODETECTION_FAILED)
+                        {
+                            _autoDetectionFailed = true;
+                            _lastTimeAutoDetectionFailed = DateTime.Now;
```

Comment:
As with earlier, we should use either Environment.TickCount or DateTime.UtcNow, but not DateTime.Now.

Reply:
I'll switch to Environment.TickCount.